### PR TITLE
show buttons for inline attachments when editing a previously saved draft

### PR DESCRIPTION
**Pull request by johnbotris**

### CODE REVIEW

File: src/mail/editor/MailEditorViewModel.js [hunk 1]
```diff
@@ -75,10 +75,9 @@ export function createPasswordField(model: SendMailModel, recipient: RecipientIn
 	}
 }
 
-export function createAttachmentButtonAttrs(model: SendMailModel, inlineImageElements: Array<HTMLElement>, mentionedInlineImages: Array<string>): Array<ButtonAttrs> {
+export function createAttachmentButtonAttrs(model: SendMailModel, inlineImageElements: Array<HTMLElement>): Array<ButtonAttrs> {
 	return model.getAttachments()
 		// Only show file buttons which do not correspond to inline images in HTML
```
Comment (charlag) [inline src/mail/editor/MailEditorViewModel.js:80]: Should comment also go?

**Review by charlag — APPROVED** [review]
I think the initial idea was that we don't show those which are from the previous place in the thread but I don't know if it makes sense even, maybe we just changed it accidentally.
Seems fine to me.

File: src/mail/editor/MailEditorViewModel.js (hunk 1 above)
Comment (johnbotris) [reply src/mail/editor/MailEditorViewModel.js:80]: comment should go...

**Comment by johnbotris** [comment]
> I think the initial idea was that we don't show those which are from the previous place in the thread

I could modify it?

**Comment by charlag** [comment]
I think it's fine as is, let's just all show em always, it's the least confusing. I'm not even sure.

**Pull request merged**